index.bs

```diff
@@ -210,6 +210,20 @@ Restrictions and Clarifications {#restrictions}
     href="https://www.w3.org/TR/CSS21/zindex.html#painting-order">the painting
     order</a> steps -- is not.
 
+* From the perspective of an <a
+    href="https://w3c.github.io/IntersectionObserver/">intersection observer</a>,
+    elements in the subtree of a [=skipped=] element are not intersecting the <a
+    href="https://w3c.github.io/IntersectionObserver/#dom-intersectionobserver-root">root</a>.
+    This is true even if both the root and the target elements are in the subtree
```

Comment:
I think this makes sense, since the target and the root are invisible in this case